CHANGELOG.asciidoc

```diff
@@ -25,7 +25,9 @@ image::https://raw.githubusercontent.com/apache/tinkerpop/master/docs/static/ima
 
 This release also includes changes from <<release-3-5-7, 3.5.7>>.
 
+==== Bugs
```

Comment:
please remove title. that's add on release

Reply:
removed at https://github.com/apache/tinkerpop/pull/2059/commits/6c5bac160f237316eabbd697522b203b05570768